cli/create.go

```diff
@@ -85,6 +89,10 @@ func runCreate(args []string) (exit int) {
 		return exitError("couldn't create swarm: --image must be provided")
 	}
 
+	if flags.ImageURI == awsEuWest1CoreOS && flags.UseIgnition {
+		return exitError(fmt.Sprintf("couldn't create swarm: --use-ignition requires a recent CoreOS AMI than '%s'", awsEuWest1CoreOS))
```

Comment:
You probably want `if flags.UseIgnition && !cmdCreate.Flags.Changed("image") {` then which feels more expressive to me for the intent.

> --use-ignition requires a recent CoreOS AMI than

How about
> ...--use-ignition requires a more recent CoreOS AMI than ...